### PR TITLE
added bg video to team, add sound to teaser video, and introduce testimonial

### PR DESCRIPTION
**Pull request by Paing-Ko**
added bg video to team, add sound to teaser video, and introduce testimonial
add some styles to about section

### CODE REVIEW

**Review by Paing-Ko — COMMENTED** [review]
nice work
when you are satisfied with the code, can you merge PR please, thanks

File: index.html [hunk 1]
```diff
@@ -50,9 +50,9 @@ <h3>Services We Provide:</h3>
                 </div>
 
                 <div class="about-img-wrapper">
-                    <img class="about-img" src="media/eco4.jpg" alt="Product Image">
```
Comment (MarikaBBB) [inline index.html]: add descriptive alt for better accessibility

File: index.html [hunk 2]
```diff
@@ -50,9 +50,9 @@ <h3>Services We Provide:</h3>
...
-                    <img class="about-img" src="media/eco5.jpg" alt="Product Image">
-                    <img class="about-img" src="media/eco3.jpg" alt="Product Image">
+                    <img class="about-img" src="media/eco4.jpg" alt="Man using VR headset in the living room, standing in front of a TV.">
+                    <img class="about-img" src="media/eco5.jpg" alt="Woman from the waist up wearing a VR headset with nature images projected onto her chest and arms.">
+                    <img class="about-img" src="media/eco3.jpg" alt="Man using a VR headset while walking on a treadmill">
```
Comment (MarikaBBB) [inline index.html:55]: Comment: The alt attributes here were too generic and did not offer a detailed description. I updated the alt attributes for better accessibility and a more precise description.

File: index.html [hunk 3]
```diff
@@ -88,7 +88,7 @@ <h2>Meet The Team</h2>
                 </div>
 
                 <div class="team-member">
-                    <img src="media/mark.jpg" alt="Makr's porttait">
+                    <img src="media/mark.jpg" alt="Makr's portrait">
```
Comment (MarikaBBB) [inline index.html:91]: Comment: Fixed typo in alt attribute.

File: index.html [hunk 4]
```diff
@@ -179,7 +179,7 @@ <h2>Contact Us</h2>
 
     </main>
 
-    <footer role="contentinfo">
```
Comment (MarikaBBB) [inline index.html:124]: Comment: Removed 'role' attribute as it is unnecessary.

File: style.css [hunk 5]
```diff
@@ -224,6 +224,7 @@ button:hover {
 
 #team {
     position: relative;
+    overflow: hidden;
```
Comment (MarikaBBB) [inline style.css:227]: Comment: Setting overflow to 'hidden' ensures that the video remains within the section boundaries.

File: style.css [hunk 6]
```diff
@@ -232,7 +233,7 @@ button:hover {
     left: 0;
     width: 100%;
     height: 100%;
-    max-height: 150vh;
+    max-height: 250vh;
```
Comment (MarikaBBB) [inline style.css:236]: Comment: Adjusted max-height to ensure the video background extends further.

File: style.css [hunk 7]
```diff
@@ -347,6 +348,8 @@ cite {
     margin-bottom: 0.5rem;
     font-weight: bold;
     color: black;
+    background-color: rgba(243, 248, 247, 0.287);
```
Comment (MarikaBBB) [inline style.css:351]: Comment: Added a background colour under the label texts for better readability.

**Review by MarikaBBB — COMMENTED** [review]
I compressed all the images to improve website performance

File: index.html (hunk 2 above)
Comment (Paing-Ko) [reply index.html:55]: thanks

File: index.html (hunk 3 above)
Comment (Paing-Ko) [reply index.html:91]: actually, it is Mark's Portrait

File: index.html (hunk 4 above)
Comment (Paing-Ko) [reply index.html:124]: thanks

**Review by Paing-Ko — COMMENTED** [review]
thanks

**Pull request merged**